### PR TITLE
Created index.js for easier use with browserify

### PR DESCRIPTION
**Pull request by barroudjo**
It will make it easier to use with browserify:

``
var myApp = require('angular').module('myApp', [ require('angular-xml') ]);

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+require('angular-xml.js');
```
Comment (johngeorgewright) [inline index.js]: Shouldn't this be `./angular-xml.js`?

Comment (barroudjo) [inline index.js]: My bad, you're right !

File: index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+module.exports = 'xml';
```
Comment (johngeorgewright) [inline index.js]: This seems odd to me, I'm no browserify expert, but in the common.js world, requiring this module would just return a string.

Comment (barroudjo) [inline index.js]: Again you're right, but now it's what's expected. This way you can use it in the list of angular modules dependencies, for example:

``
var myApp = require('angular').module('myApp', [ require('angular-xml') ]);
``

It's a standard that's been adopted among major AngularJS modules (for [example angular-loading-bar](https://github.com/chieffancypants/angular-loading-bar/blob/master/index.js))

Comment (johngeorgewright) [inline index.js]: OK, cool. I'll have a read up later.

**Comment by johngeorgewright** [comment]
If you can make that small change and squash the commits, then I'll package up a release.

**Comment by barroudjo** [comment]
I don't know what you mean by squash the commits. I commited the correction for the ./angular-xml.

**Comment by johngeorgewright** [comment]
Squashed and merged

**Pull request closed without merging**